docs/iidm/model/threeWindingsTransformer.md

```diff
@@ -5,23 +5,34 @@ layout: default
 
 The `com.powsybl.iidm.network.ThreeWindingsTransformer` interface is used to model a three windings power transformer.
 A three windings power transformer is connected to three voltage levels (side 1, side 2 and side 3) that belong to the
-same substation:
+same substation, usually:
```

Comment:
Why "usually"? Is it possible? Should it be indicated, keeping in mind it is forbidden in IIDM and we are describing the IIDM model?

Reply:
In IIDM there was no explicit check on the order of the legs. In the previous documentation, it was stated that side 1 should correspond to primary, and side 2 and 3 were interchangeable, but this was not enforced in the code.

We have written the documentation according to the current code. There is no check on the values of the rated or base voltages of each side.

In CGMES, there is a rule about the order of transformer ends (1-primary, 2-secondary, 3-tertiary), and this is respected when building the IIDM legs.

Reply:
Okay, I misunderstood this bit: as it was written, I thought it meant a t3w was connected to 3 voltage levels that were *usually* in the same substation but not always, this was why I was confused...

Maybe reword the sentence as below:
`A three windings transformer [...] to the same substation. We usually have:` 
`- side 1 as the primary side with the highest voltage`
`- side 2 as the secondary side with the medium voltage`
`- side 3 as the tertiary side with the lowest voltage`

Reply:
done